tests/components/homematicip_cloud/helper.py

```diff
@@ -23,26 +30,36 @@ def get_and_check_entity_basics(
     """Get and test basic device."""
     ha_entity = hass.states.get(entity_id)
```

Comment:
The return value here is a state, and not an entity. I suggest we rename it to reflect that.

Reply:
i replace it in all occurances